gpflow/optimizers/mcmc.py

```diff
@@ -22,7 +22,7 @@
 
 
 ModelParameters = Sequence[TypeVar("ModelParameter", tf.Variable, Parameter)]
-LogProbabilityFunction = Callable[[ModelParameters], tf.Tensor]
+LogProbabilityFunction = Callable[[], tf.Tensor]  # implicitly a function of ModelParameters
```

Comment:
is this comment needed?

Reply:
removed it (the whole line will be removed by #1374)